go-controller/pkg/clustermanager/egressservice/egressservice_cluster_endpointslices.go

```diff
@@ -61,6 +62,10 @@ func (c *Controller) onEndpointSliceDelete(obj interface{}) {
 }
 
 func (c *Controller) queueServiceForEndpointSlice(endpointSlice *discovery.EndpointSlice) {
+	if util.IsMirrorEndpointSlice(endpointSlice) {
```

Comment:
Maybe add a comment in all these if statements for egress service to say // TODO: Add support for UDN with Egress Service

Reply:
Added a TODO for EgressServices